statsmodels/genmod/families/family.py

```diff
@@ -671,13 +730,14 @@ def loglike(self, endog, mu, freq_weights=1., scale=1.):
         -------
         llf : float
             The value of the loglikelihood function evaluated at
-            (endog,mu,scale) as defined below.
+            (endog,mu,freq_weights,scale) as defined below.
 
         Notes
         --------
-        llf = -1/scale * sum(endog/mu + log(mu) + (scale-1)*log(endog) +\
-              log(scale) + scale*gammaln(1/scale))
-        where gammaln is the log gamma function.
+        .. math::
+
+           llf = -1 / \phi * \sum_i freq\_weights_i * (Y_i / \mu_i+\log(\mu_i)+
+           (\phi -1) * \log(Y) + \log(\phi) + \phi * \ln \Gamma(1 /\phi))
```

Comment:
what's `\phi` here
in general I don't like many of the Greek letter (personal taste and not mnenomic)
needs at least a note/definition, e.g. where `$\phi$ is ... and `$\mu` is ...

`mu` for conditional expectation/mean might be the only Greek letter left in the code. (I suggested kicking them out as much as possible)